addons/mail/static/src/core/common/chat_window_model.js

```diff
@@ -70,7 +70,7 @@ export class ChatWindow extends Record {
         this.notifyState();
     }
 
-    open({ notifyState = true } = {}) {
+    open({ notifyState = true, shouldFocus = true } = {}) {
```

Comment:
I would inverse the default value, focusing should always be an active action, purposely chosen.